.fix-esm-imports.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Search for "from './baz'" and transform to "from './baz.js'"
+find dist/esm -type f -name *.js -exec sed -i -e -E "s/from '(\.\/.*)'/from '\1.js'/g" {} \;
```

Comment:
Tested on debian the `-e` parameter doesn't work.

Reply:
also this won't catch any pure imports like `import 'utils/myfile.js'`

Reply:
Note for anyone else viewing this that I tried getting a rule into eslint that would resolve this for similar use-cases: https://github.com/import-js/eslint-plugin-import/pull/2701